### PR TITLE
LPS-187773 Sort all Objects entities by label by default

### PR DESCRIPTION
**Pull request by pedro-oliveira446**
Tasks:
[LPS-188374](https://liferay.atlassian.net/browse/LPS-188374)
[LPS-188375](https://liferay.atlassian.net/browse/LPS-188375)
[LPS-188376](https://liferay.atlassian.net/browse/LPS-188376)
[LPS-188377](https://liferay.atlassian.net/browse/LPS-188377)

### CODE REVIEW

**Comment by pedro-oliveira446** [comment]
ci:test:sf

**Comment by pedro-oliveira446** [comment]
ci:test:relevant

**Comment by pedro-oliveira446** [comment]
ci:test:object

**Review by p-albuquerque — APPROVED** [review]

**Review by MarinhoFeliphe — APPROVED** [review]
LGTM!! :tada:

**Comment by MarinhoFeliphe** [comment]
@pedro-oliveira446 solve the conflicts here as well,

Thank you!

**Review by alinedoleron — APPROVED** [review]
LGTM!

**Comment by pedro-oliveira446** [comment]
ci:test:sf

**Comment by pedro-oliveira446** [comment]
ci:test:relevant

**Comment by pedro-oliveira446** [comment]
ci:test:object

**Comment by pedro-oliveira446** [comment]
ci:test:sf

**Comment by pedro-oliveira446** [comment]
ci:test:relevant

**Comment by pedro-oliveira446** [comment]
ci:test:object

**Comment by pedro-oliveira446** [comment]
Hi @MarinhoFeliphe, @p-albuquerque,

I had to add four new commits to fix the related integration tests. Can you review again, please ?

**Comment by mateussgn** [comment]
I'm reviewing this PR 🙂

**Review by mateussgn — APPROVED** [review]

**Comment by mateussgn** [comment]
LGTM! 🎉

**Comment by mateussgn** [comment]
Hey, @guilhermedcamacho, can double-check this PR review?

File: modules/apps/object/object-admin-rest-test/src/testIntegration/java/com/liferay/object/admin/rest/resource/v1_0/test/ObjectValidationRuleResourceTest.java [hunk 1]
```diff
@@ -212,6 +235,56 @@ protected ObjectValidationRule randomObjectValidationRule()
 				randomObjectValidationRule());
 	}
 
+	private void _unsafeTriConsumer(
```
Comment (MarinhoFeliphe) [inline modules/apps/object/object-admin-rest-test/src/testIntegration/java/com/liferay/object/admin/rest/resource/v1_0/test/ObjectValidationRuleResourceTest.java:238]: @pedro-oliveira446 rename to `_getPageWithSortStringUnsafeTriConsumer` methods should be named as actions.

Thanks!

File: modules/apps/object/object-admin-rest-test/src/testIntegration/java/com/liferay/object/admin/rest/resource/v1_0/test/ObjectFieldResourceTest.java [hunk 2]
```diff
@@ -422,6 +584,48 @@ private ObjectField _addObjectField() throws Exception {
 		return _objectField;
 	}
 
+	private void _unsafeTriConsumer(
```
Comment (MarinhoFeliphe) [inline modules/apps/object/object-admin-rest-test/src/testIntegration/java/com/liferay/object/admin/rest/resource/v1_0/test/ObjectFieldResourceTest.java:587]: same here

**Review by MarinhoFeliphe — CHANGES_REQUESTED** [review]

**Comment by pedro-oliveira446** [comment]
ci:test:sf

**Comment by pedro-oliveira446** [comment]
ci:test:relevant

**Comment by pedro-oliveira446** [comment]
ci:test:object

**Review by MarinhoFeliphe — APPROVED** [review]
LGTM!! :tada:

**Comment by MarinhoFeliphe** [comment]
ci:forward:force

**Pull request closed without merging**